run_tests.sh

```diff
@@ -9,6 +9,9 @@ if [ "$1" == "distrib" ] ; then
   cd ../../
 fi
 
+eval `opam config env`
+bap-server &
```

Comment:
is this necessary to run QIRA. if so, it needs to be added to both install and the main QIRA program